### DISCUSSION TITLE
Community liaison / outreach

### DISCUSSION THREAD

**Discussion by monochromec** [Homebrew/discussions#5689] [Everyday usage] [answered]
Hi Mike/John,

How would I get in touch with a community liaison/manager (if existing) of the project?

   Cheers, Chris

**Comment by MikeMcQuaid** [comment, marked as answer]
@monochromec Hi! I'm the Project Leader of Homebrew and, which I think you may be referring to be mentioning John (CEO) here, I'm also the CTO and Cofounder of Workbrew. If this is about Homebrew: happy to keep talking here. If it's about Workbrew: could you please contact us at https://workbrew.com/contact or an existing point of contact? Thanks!

**Comment by monochromec** [comment]
Left you  message via the contact sheet yesterday. Please let me know if you got this.

**Reply by MikeMcQuaid** [reply]
@monochromec can you confirm what name/organisation you used there?

**Comment by monochromec** [comment]
It's Linux Inlaws (linuxinlaws.eu).

**Reply by MikeMcQuaid** [reply]
@monochromec We haven't seen anything from you, sorry. Can you please email help@workbrew.com or john@workbrew.com and CC mike@workbrew.com? Thanks.

**Comment by monochromec** [comment]
On its way...

**Reply by MikeMcQuaid** [reply]
Got it now, thanks.
